lib/airbrake.js

```diff
@@ -46,7 +47,7 @@ function Airbrake() {
   ];
 }
 
-_.assign(Airbrake.prototype, EventEmitter.prototype);
+assign(Airbrake.prototype, EventEmitter.prototype);
```

Comment:
I believe assign is more performant as it doesn't deep merge. I don't really know though, I didn't want to change any of the logic in this PR. You could also use Object.assign(), but that is only available in Node v4 and later.